src/main/java/net/wurstclient/hacks/NoSlowdownHack.java

```diff
@@ -10,14 +10,53 @@
 import net.wurstclient.Category;
 import net.wurstclient.SearchTags;
 import net.wurstclient.hack.Hack;
+import net.wurstclient.settings.CheckboxSetting;
 
 @SearchTags({"no slowdown", "no slow down"})
 public final class NoSlowdownHack extends Hack
 {
+	private final CheckboxSetting noIce = new CheckboxSetting(
+			"No ice", "Prevent ice slipperiness\n",
+			false);
+	
+	private final CheckboxSetting honeyJump = new CheckboxSetting(
+			"Honey jump", "Allow jumping on honey\n",
+			true);
+	
+	private final CheckboxSetting slimeUtil = new CheckboxSetting(
+			"Slime utiltiy", "Removes bouncing and disables slowdown\n"
+					+ "\u00A74Don't disable this while on a slime block\u00A7r\n"
+					+ "Slime still negates falldamage",
+			false);
+	
 	public NoSlowdownHack()
 	{
 		super("NoSlowdown", "Cancels slowness effects caused by\n"
 			+ "honey, soul sand and using items.");
 		setCategory(Category.MOVEMENT);
+		addSetting(honeyJump);
+		addSetting(noIce);
+		addSetting(slimeUtil);
+	}
+	
+	public boolean getNoIce()
+	{
+	if (noIce.isChecked())
+		return true;
+	        return false;
+	}
+	
+	public boolean getHoneyJump()
+	{
+	if (honeyJump.isChecked())
+		return true;
+		return false;
+	}
+	
+	public boolean getSlimeUtil()
+	{
+	if (slimeUtil.isChecked())
+		return true;
+		return false;

```

Comment:
```suggestion
	public boolean getNoIce()
	{
		return noIce.isChecked();
	}
	
	public boolean getHoneyJump()
	{
		return honeyJump.isChecked();
	}
	
	public boolean getSlimeUtil()
	{
		return slimeUtil.isChecked();
```

Reply:
Hey, thank you for the suggestion, I'm pretty new to coding so these things happen. Thank you for opening my eyes to a simpler method!